WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -21,13 +21,15 @@ public class ReaderScreen: ScreenObject {
         )
     }
 
-    public func openLastPost() {
+    public func openLastPost() throws -> ReaderScreen {
```

Comment:
I don't see any `try` in the implementation. Is this `throws` necessary? 

```suggestion
    public func openLastPost() -> ReaderScreen {
```